src/vma/util/sys_vars.cpp

```diff
@@ -621,6 +622,16 @@ void mce_sys_var::get_env_params()
 
 	read_hv();
 
+	if((env_ptr = getenv(SYS_VAR_SOCKETXTREME )) != NULL) {
+		enable_socketxtreme = atoi(env_ptr) ? true : false;
+	}
+	if (enable_socketxtreme) {
```

Comment:
I am expecting issues once we will add VMA_SOCKETXTRME to any spec...

Reply:
It will be harmful decision because setting socketxtreme mode requires form the user application be prepared for this.

Reply:
it there an option to fix the logic so it will not happen ?
in any case we should add some explanation that this check should be the first check.